ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -297,22 +302,64 @@ private class SqlLedger(
         case Failure(f) => Failure(f)
       }(DEC)
 
+  //TODO BH: revisit this and where it is needed
   override def allocateParty(
       party: Party,
-      displayName: Option[String]): Future[PartyAllocationResult] =
-    ledgerDao
+      displayName: Option[String],
+      submissionId: String,
+      participantId: ParticipantId): Future[SubmissionResult] = {
+
+    val storePartyAllocationEntry: Future[SubmissionResult] = {
+      var headRef = 0L
```

Comment:
Party allocation entries need to go through the same "enqueuing" mechanism as the transaction ledger entries.

Reply:
enqueue now there on the branch.  NOTE: i had to reduce the number of parties in the LotsOfParties test to 125 as the previoius 1024 were failing in the two LotsOfParties tests throwing a mess of RESOURCE_EXHAUSTED grpc overload errors.

spoke with @stefanobaghino-da this morning and said we will temporarily reduce to get this merged in, but will need to investigate why this behaves differently when using enqueue, and pump back up to higher number once we have found the issue and resolved in a separate PR